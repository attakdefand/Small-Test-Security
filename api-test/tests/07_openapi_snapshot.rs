use insta::assert_json_snapshot;

#[test]
fn openapi_stable() {
    // Replace with your real exporter, e.g., app::openapi()
    let spec = serde_json::json!({"openapi":"3.0.3","paths":{r"/health":{}}});
    assert_json_snapshot!("openapi.json", spec);
}