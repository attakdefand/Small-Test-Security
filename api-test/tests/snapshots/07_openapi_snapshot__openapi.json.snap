---
source: api-test/tests/07_openapi_snapshot.rs
expression: spec
---
{
  "openapi": "3.0.3",
  "paths": {
    "/health": {}
  }
}
